examples/TensorFlow_ClusterSpec/README.md

```diff
@@ -0,0 +1,132 @@
+is used, so the python binary has to be defined as follows:
+
+```
+./venv/bin/python
+```
+
+**Summary**
+
+Finally, full commands required to launch the training in our exemplary environment will be as follows:
+
+worker on 172.17.0.3:
+```bash
+export LD_LIBRARY_PATH=native_client/
+./venv/bin/python ./DeepSpeech.py --train_files=ldc93s1/ldc93s1.csv --dev_files=ldc93s1/ldc93s1.csv --test_files=ldc93s1/ldc93s1.csv --ps_hosts=172.17.0.3:2224 --worker_hosts=172.17.0.3:2222,172.17.0.4:2223 --job_name=worker --task_index=0
+```
+
+worker on 172.17.0.4: 
+```bash
+export LD_LIBRARY_PATH=native_client/
+./venv/bin/python ./DeepSpeech.py --train_files=ldc93s1/ldc93s1.csv --dev_files=ldc93s1/ldc93s1.csv --test_files=ldc93s1/ldc93s1.csv --ps_hosts=172.17.0.3:2224 --worker_hosts=172.17.0.3:2222,172.17.0.4:2223 --job_name=worker --task_index=1
+```
+
+parameter server on 172.17.0.3:
+```bash
+export LD_LIBRARY_PATH=native_client/
+./venv/bin/python ./DeepSpeech.py --train_files=ldc93s1/ldc93s1.csv --dev_files=ldc93s1/ldc93s1.csv --test_files=ldc93s1/ldc93s1.csv --ps_hosts=172.17.0.3:2224 --worker_hosts=172.17.0.3:2222,172.17.0.4:2223 --job_name=ps --task_index=0
+```
+
+## Running the training with TensorHive
+
+The TensorHive `task execution` module allows convenient orchestration of distributed trainings.
+It is available in the Tasks Overview view. The `CREATE TASKS FROM TEMPLATE` button allows to
+conveniently configure tasks supporting a specific framework or distribution method. In this
+example we choose the Tensorflow - cluster parameters template, and click `GO TO TASK CREATOR`:
+
+![choose_template](https://raw.githubusercontent.com/roscisz/TensorHive/master/examples/TensorFlow_ClusterSpec/img/choose_template.png)
+
+In the task creator, we set the Command to
+```
+./venv/bin/python ./DeepSpeech.py --train_files=ldc93s1/ldc93s1.csv --dev_files=ldc93s1/ldc93s1.csv --test_files=ldc93s1/ldc93s1.csv
+```
+
+In order to add the LD_LIBRARY_PATH environment variable, we enter the parameter name,
+select Static (the same value for all processes) and click `ADD AS ENV VARIABLE TO ALL TASKS`.
+Then, set the appropriate value of the environment variable (native_client/):
+
+
+![env_var](https://raw.githubusercontent.com/roscisz/TensorHive/master/examples/TensorFlow_ClusterSpec/img/env_var.png)
+
+
+The task creator allows also to conveniently specify other command-line arguments. For example,
+should the `train_files`, `dev_files` and `test_files` parameters change throughout the training
+processes, they could be handled by `ADD AS PARAMETER TO ALL TASKS`.
+
+Use `ADD TASK` to create as many copies of the defined process as required and
+select the appropriate hostname and resource (CPU/GPU_N) for the specified training process. The resultant
+command that will be executed by TensorHive on the selected node will be displayed above the process specification.
+Note that the cluster parameters and CUDA_VISIBLE_DEVICES variable are configured automatically:
```

Comment:
Don't we have to change parameter server process `job_name` parameter value to `ps` manually?

Reply:
indeed, added description in bae71bdd663852f8e34230adb395e6b12910ca02